### DISCUSSION TITLE
Background defaults to purple?

### DISCUSSION THREAD

**Discussion by bryiac** [system-ui/theme-ui#1270] [Q&A] [answered]
When you don't explicitly set a background it defaults to purple is there any reason for this?

**Comment by hasparus** [comment, marked as answer]
So... there are no default values, and the thing that lands in your CSS is actually `background-color: background`. 
It's just "unsupported" on Theme UI side. I think the library expects you to have a background color.

As of why is `background` `rgb(99, 99, 206)`... I tried googling it, but I didn't find any explanation.

**Reply by bryiac** [reply]
that's really weird I honestly thought it was theme-ui making it purple thanks for the response

**Reply by hasparus** [reply]
We have to use the key if we don't find the value, because we don't prefix design tokens in any way, and instead of `"background"` you could pass `"blue"`, which could be both an HTML color or your theme color.

This is what happens with the default body style... and it turns out that `background` is purple.
